htop fails to build on amd64

The package htop with version 3.4.1 and epoch 1 fails to build from source on amd64 with the following error:

```
2025-06-24 08:58:58.929051349 +0000 UTC parsed env
2025-06-24 08:58:58.92925669 +0000 UTC configuring puller identity "720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69"...
2025-06-24 08:58:58.92970359 +0000 UTC running command chainctl [auth login --audience apk.cgr.dev --identity 720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69]
2025-06-24 08:58:59.31898389 +0000 UTC Successfully exchanged token.
2025-06-24 08:58:59.486886227 +0000 UTC Valid! Id: 720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69
2025-06-24 08:58:59.542144445 +0000 UTC Updates are available for chainctl (current version: 0.2.91; latest: 0.2.92). To install, please run:
2025-06-24 08:58:59.542148031 +0000 UTC     $ chainctl update
2025-06-24 08:58:59.5452223 +0000 UTC command "chainctl" completed successfully
2025-06-24 08:58:59.545296401 +0000 UTC puller identity configured successfully
2025-06-24 08:58:59.54530614 +0000 UTC puller identity configured successfully
2025-06-24 08:58:59.545312634 +0000 UTC running build...
2025-06-24 08:58:59.545323627 +0000 UTC running command melange [build htop.yaml --gcplog --source-dir htop --arch x86_64 --arch=x86_64 --env-file=build-x86_64.env --namespace=wolfi --pipeline-dir=pipelines --out-dir=./packages/ --runner=qemu --keyring-append=https://packages.wolfi.dev/os/wolfi-signing.rsa.pub --repository-append=https://packages.wolfi.dev/os --repository-append=https://apk.cgr.dev/wolfi-presubmit/world-1750746723102 --repository-append=https://apk.cgr.dev/wolfi-presubmit/world-1750746723102 --git-repo-url=https://github.com/wolfi-dev/os]
2025-06-24 08:58:59.562788836 +0000 UTC melange v0.26.12+dirty is building:
2025-06-24 08:58:59.566966482 +0000 UTC populating workspace /tmp/melange-workspace-866924975 from htop
2025-06-24 08:58:59.659892938 +0000 UTC Updates are available for chainctl (current version: 0.2.91; latest: 0.2.92). To install, please run:
2025-06-24 08:58:59.659896514 +0000 UTC     $ chainctl update
2025-06-24 08:59:01.243963996 +0000 UTC image configuration:
2025-06-24 08:59:01.244041756 +0000 UTC   contents:
2025-06-24 08:59:01.244052278 +0000 UTC     build repositories: [https://apk.cgr.dev/wolfi-presubmit/world-1750746723102 https://packages.wolfi.dev/os]
2025-06-24 08:59:01.244072271 +0000 UTC     runtime repositories: []
2025-06-24 08:59:01.244081575 +0000 UTC     keyring:      [https://packages.wolfi.dev/os/wolfi-signing.rsa.pub]
2025-06-24 08:59:01.244090865 +0000 UTC     packages:     [autoconf=2.72-r3 automake=1.18-r1 binutils=2.44-r9 build-base=1-r8 busybox=1.37.0-r46 ca-certificates-bundle=20241121-r42 gcc=15.1.0-r1 glibc-dev=2.41-r50 glibc-locale-posix=2.41-r50 glibc=2.41-r50 gmp=6.3.0-r7 isl=0.27-r3 ld-linux=2.41-r50 libatomic=15.1.0-r1 libbz2-1=1.0.8-r18 libcap-dev=2.76-r4 libcap-utils=2.76-r4 libcap=2.76-r4 libcrypt1=2.41-r50 libcrypto3=3.5.0-r3 libgcc=15.1.0-r1 libgo=15.1.0-r1 libgomp=15.1.0-r1 libnl3-cli=3.11.0-r4 libnl3-dev=3.11.0-r4 libnl3=3.11.0-r4 libquadmath=15.1.0-r1 libssl3=3.5.0-r3 libstdc++-dev=15.1.0-r1 libstdc++=15.1.0-r1 libxcrypt-dev=4.4.38-r2 libxcrypt=4.4.38-r2 libzstd1=1.5.7-r2 linux-headers=6.15.3-r0 m4=1.4.20-r4 make=4.4.1-r7 mpc=1.3.1-r6 mpfr=4.2.2-r1 ncurses-dev=6.5_p20241228-r3 ncurses-terminfo-base=6.5_p20241228-r3 ncurses=6.5_p20241228-r3 nss-db=2.41-r50 nss-hesiod=2.41-r50 openssf-compiler-options=20240627-r21 perl=5.40.2-r1 pkgconf-dev=2.5.0-r0 pkgconf=2.5.0-r0 posix-cc-wrappers=1-r6 scanelf=1.3.8-r4 wget=1.25.0-r2 wolfi-baselayout=20230201-r21 zlib=1.3.1-r50]
2025-06-24 08:59:01.244131658 +0000 UTC   accounts:
2025-06-24 08:59:01.244144976 +0000 UTC     runas:  
2025-06-24 08:59:01.244158308 +0000 UTC     users:
2025-06-24 08:59:01.244169274 +0000 UTC       - uid=1000(build) gid=1000
2025-06-24 08:59:01.244184853 +0000 UTC     groups:
2025-06-24 08:59:01.244199707 +0000 UTC       - gid=1000(build) members=[build]
2025-06-24 08:59:01.244213059 +0000 UTC auth configured for: []
2025-06-24 08:59:01.743000308 +0000 UTC installing libgcc (15.1.0-r1)
2025-06-24 08:59:01.743573618 +0000 UTC installing wolfi-baselayout (20230201-r21)
2025-06-24 08:59:01.770775574 +0000 UTC installing ca-certificates-bundle (20241121-r42)
2025-06-24 08:59:01.771135558 +0000 UTC installing glibc-locale-posix (2.41-r50)
2025-06-24 08:59:01.780440066 +0000 UTC installing glibc (2.41-r50)
2025-06-24 08:59:01.781086708 +0000 UTC installing ld-linux (2.41-r50)
2025-06-24 08:59:01.781620207 +0000 UTC installing libbz2-1 (1.0.8-r18)
2025-06-24 08:59:01.781976737 +0000 UTC installing libxcrypt (4.4.38-r2)
2025-06-24 08:59:01.782331057 +0000 UTC installing libcrypt1 (2.41-r50)
2025-06-24 08:59:01.783398732 +0000 UTC installing zlib (1.3.1-r50)
2025-06-24 08:59:02.219153315 +0000 UTC installing perl (5.40.2-r1)
2025-06-24 08:59:02.249195283 +0000 UTC installing m4 (1.4.20-r4)
2025-06-24 08:59:02.249655057 +0000 UTC installing autoconf (2.72-r3)
2025-06-24 08:59:02.249971735 +0000 UTC installing automake (1.18-r1)
2025-06-24 08:59:02.250362124 +0000 UTC installing libstdc++ (15.1.0-r1)
2025-06-24 08:59:02.250715022 +0000 UTC installing libzstd1 (1.5.7-r2)
2025-06-24 08:59:02.53158212 +0000 UTC installing binutils (2.44-r9)
2025-06-24 08:59:02.532437853 +0000 UTC installing libquadmath (15.1.0-r1)
2025-06-24 08:59:02.532781788 +0000 UTC installing posix-cc-wrappers (1-r6)
2025-06-24 08:59:02.53293694 +0000 UTC installing libatomic (15.1.0-r1)
2025-06-24 08:59:02.533238962 +0000 UTC installing gmp (6.3.0-r7)
2025-06-24 08:59:02.533463401 +0000 UTC installing libgo (15.1.0-r1)
2025-06-24 08:59:02.53375936 +0000 UTC installing libgomp (15.1.0-r1)
2025-06-24 08:59:02.534045385 +0000 UTC installing isl (0.27-r3)
2025-06-24 08:59:02.534326711 +0000 UTC installing mpfr (4.2.2-r1)
2025-06-24 08:59:02.5346412 +0000 UTC installing mpc (1.3.1-r6)
2025-06-24 08:59:02.534926067 +0000 UTC installing libxcrypt-dev (4.4.38-r2)
2025-06-24 08:59:02.535255069 +0000 UTC installing nss-db (2.41-r50)
2025-06-24 08:59:02.535731874 +0000 UTC installing nss-hesiod (2.41-r50)
2025-06-24 08:59:02.536168276 +0000 UTC installing linux-headers (6.15.3-r0)
2025-06-24 08:59:02.538029394 +0000 UTC installing glibc-dev (2.41-r50)
2025-06-24 08:59:02.539200987 +0000 UTC installing libstdc++-dev (15.1.0-r1)
2025-06-24 08:59:02.541193314 +0000 UTC installing openssf-compiler-options (20240627-r21)
2025-06-24 08:59:06.793884524 +0000 UTC installing gcc (15.1.0-r1)
2025-06-24 08:59:06.795756343 +0000 UTC installing make (4.4.1-r7)
2025-06-24 08:59:06.796147954 +0000 UTC installing pkgconf (2.5.0-r0)
2025-06-24 08:59:06.796514532 +0000 UTC installing build-base (1-r8)
2025-06-24 08:59:06.79666724 +0000 UTC installing busybox (1.37.0-r46)
2025-06-24 08:59:06.79700657 +0000 UTC installing libcap (2.76-r4)
2025-06-24 08:59:06.797320536 +0000 UTC installing libcap-utils (2.76-r4)
2025-06-24 08:59:06.797636285 +0000 UTC installing libcap-dev (2.76-r4)
2025-06-24 08:59:06.797943594 +0000 UTC installing libcrypto3 (3.5.0-r3)
2025-06-24 08:59:06.798315836 +0000 UTC installing libnl3 (3.11.0-r4)
2025-06-24 08:59:06.798686449 +0000 UTC installing libnl3-cli (3.11.0-r4)
2025-06-24 08:59:06.799105304 +0000 UTC installing libnl3-dev (3.11.0-r4)
2025-06-24 08:59:06.799683159 +0000 UTC installing libssl3 (3.5.0-r3)
2025-06-24 08:59:06.800067349 +0000 UTC installing ncurses-terminfo-base (6.5_p20241228-r3)
2025-06-24 08:59:06.80070154 +0000 UTC installing ncurses (6.5_p20241228-r3)
2025-06-24 08:59:06.801093976 +0000 UTC installing ncurses-dev (6.5_p20241228-r3)
2025-06-24 08:59:06.80150897 +0000 UTC installing pkgconf-dev (2.5.0-r0)
2025-06-24 08:59:06.801880957 +0000 UTC installing scanelf (1.3.8-r4)
2025-06-24 08:59:06.802133556 +0000 UTC installing wget (1.25.0-r2)
2025-06-24 08:59:11.556661532 +0000 UTC qemu: generating ssh key pairs for ephemeral VM
2025-06-24 08:59:11.557818857 +0000 UTC qemu: generating initramfs
2025-06-24 08:59:11.826127642 +0000 UTC image configuration:
2025-06-24 08:59:11.826184092 +0000 UTC   contents:
2025-06-24 08:59:11.82619163 +0000 UTC     build repositories: [https://apk.cgr.dev/chainguard]
2025-06-24 08:59:11.826202786 +0000 UTC     runtime repositories: []
2025-06-24 08:59:11.826208099 +0000 UTC     keyring:      []
2025-06-24 08:59:11.82621306 +0000 UTC     packages:     [microvm-init]
2025-06-24 08:59:13.766239102 +0000 UTC installing wolfi-baselayout (20230201-r21)
2025-06-24 08:59:13.767723288 +0000 UTC installing ca-certificates-bundle (20241121-r42)
2025-06-24 08:59:13.768537064 +0000 UTC installing libgcc (15.1.0-r1)
2025-06-24 08:59:13.769152536 +0000 UTC installing glibc-locale-posix (2.41-r50)
2025-06-24 08:59:13.871470447 +0000 UTC installing glibc (2.41-r50)
2025-06-24 08:59:13.876036722 +0000 UTC installing ld-linux (2.41-r50)
2025-06-24 08:59:13.877032123 +0000 UTC installing gnutar (1.35-r4)
2025-06-24 08:59:13.882506137 +0000 UTC installing libattr1 (2.5.2-r50)
2025-06-24 08:59:13.883127681 +0000 UTC installing attr (2.5.2-r50)
2025-06-24 08:59:13.884059684 +0000 UTC installing zlib (1.3.1-r50)
2025-06-24 08:59:13.884752627 +0000 UTC installing libzstd1 (1.5.7-r2)
2025-06-24 08:59:13.885978693 +0000 UTC installing xz (5.8.1-r1)
2025-06-24 08:59:13.965776779 +0000 UTC installing libcrypto3 (3.5.0-r3)
2025-06-24 08:59:13.969294408 +0000 UTC installing kmod (34.2-r40)
2025-06-24 08:59:13.970376389 +0000 UTC installing libbz2-1 (1.0.8-r18)
2025-06-24 08:59:13.971006765 +0000 UTC installing libelf (0.193-r1)
2025-06-24 08:59:13.972163002 +0000 UTC installing libmnl (1.0.5-r6)
2025-06-24 08:59:13.972748122 +0000 UTC installing libnftnl (1.2.9-r0)
2025-06-24 08:59:13.973858275 +0000 UTC installing iptables (1.8.11-r22)
2025-06-24 08:59:13.979853184 +0000 UTC installing iproute2 (6.15.0-r0)
2025-06-24 08:59:13.982868339 +0000 UTC installing libblkid (2.41-r42)
2025-06-24 08:59:13.984492242 +0000 UTC installing libmount (2.41-r42)
2025-06-24 08:59:13.986111712 +0000 UTC installing mount (2.41-r42)
2025-06-24 08:59:13.986765143 +0000 UTC installing ncurses-terminfo-base (6.5_p20241228-r3)
2025-06-24 08:59:13.995653833 +0000 UTC installing ncurses (6.5_p20241228-r3)
2025-06-24 08:59:13.997417698 +0000 UTC installing setarch (2.41-r42)
2025-06-24 08:59:13.998149344 +0000 UTC installing libuuid (2.41-r42)
2025-06-24 08:59:13.998778998 +0000 UTC installing libfdisk (2.41-r42)
2025-06-24 08:59:14.000377768 +0000 UTC installing sqlite-libs (3.50.1-r0)
2025-06-24 08:59:14.001862748 +0000 UTC installing util-linux (2.41-r42)
2025-06-24 08:59:14.005925801 +0000 UTC installing libsmartcols (2.41-r42)
2025-06-24 08:59:14.007256619 +0000 UTC installing util-linux-misc (2.41-r42)
2025-06-24 08:59:14.020006339 +0000 UTC installing libpcre2-8-0 (10.45-r2)
2025-06-24 08:59:14.021000171 +0000 UTC installing libsepol (3.8.1-r50)
2025-06-24 08:59:14.022563805 +0000 UTC installing libselinux (3.8.1-r50)
2025-06-24 08:59:14.031989194 +0000 UTC installing libxcrypt (4.4.38-r2)
2025-06-24 08:59:14.032658638 +0000 UTC installing libcrypt1 (2.41-r50)
2025-06-24 08:59:14.033259134 +0000 UTC installing linux-pam (1.7.1-r0)
2025-06-24 08:59:14.0426218 +0000 UTC installing openssh-keygen (10.0_p1-r1)
2025-06-24 08:59:14.043464246 +0000 UTC installing openssh-server-config (10.0_p1-r1)
2025-06-24 08:59:14.043969775 +0000 UTC installing openssh-server (10.0_p1-r1)
2025-06-24 08:59:14.046167986 +0000 UTC installing libstdc++ (15.1.0-r1)
2025-06-24 08:59:14.048481659 +0000 UTC installing inih (60-r0)
2025-06-24 08:59:14.049086935 +0000 UTC installing userspace-rcu (0.15.3-r0)
2025-06-24 08:59:14.051765659 +0000 UTC installing xfsprogs-core (6.14.0-r4)
2025-06-24 08:59:14.053611793 +0000 UTC installing xfsprogs (6.14.0-r4)
2025-06-24 08:59:14.054096872 +0000 UTC installing busybox (1.37.0-r46)
2025-06-24 08:59:14.055102046 +0000 UTC installing microvm-init (0.0.1-r11)
2025-06-24 08:59:14.961723168 +0000 UTC qemu: starting VM
2025-06-24 08:59:14.962845385 +0000 UTC qemu: waiting for SSH
2025-06-24 08:59:18.17624951 +0000 UTC qemu: meta-data=/dev/vda               isize=512    agcount=8, agsize=1638400 blks
2025-06-24 08:59:18.176376216 +0000 UTC qemu:          =                       sectsz=4096  attr=2, projid32bit=1
2025-06-24 08:59:18.176385824 +0000 UTC qemu:          =                       crc=1        finobt=1, sparse=1, rmapbt=1
2025-06-24 08:59:18.176393766 +0000 UTC qemu:          =                       reflink=1    bigtime=1 inobtcount=1 nrext64=1
2025-06-24 08:59:18.176400033 +0000 UTC qemu:          =                       exchange=0   metadir=0
2025-06-24 08:59:18.17641579 +0000 UTC qemu: data     =                       bsize=4096   blocks=13107200, imaxpct=25
2025-06-24 08:59:18.176425077 +0000 UTC qemu:          =                       sunit=0      swidth=0 blks
2025-06-24 08:59:18.176429914 +0000 UTC qemu: naming   =version 2              bsize=4096   ascii-ci=0, ftype=1, parent=0
2025-06-24 08:59:18.176477422 +0000 UTC qemu: log      =internal log           bsize=4096   blocks=16384, version=2
2025-06-24 08:59:18.176520739 +0000 UTC qemu:          =                       sectsz=4096  sunit=1 blks, lazy-count=1
2025-06-24 08:59:18.176526797 +0000 UTC qemu: realtime =none                   extsz=4096   blocks=0, rtextents=0
2025-06-24 08:59:18.176537293 +0000 UTC qemu:          =                       rgcount=0    rgsize=0 extents
2025-06-24 08:59:18.177521147 +0000 UTC qemu: Discarding blocks...Done.
2025-06-24 08:59:19.23313203 +0000 UTC qemu: ssh-keygen: generating new host keys: RSA ECDSA ED25519 
2025-06-24 08:59:19.240142116 +0000 UTC qemu: Server listening on 0.0.0.0 port 2223.
2025-06-24 08:59:19.240207906 +0000 UTC qemu: Server listening on 0.0.0.0 port 22.
2025-06-24 08:59:19.984976613 +0000 UTC qemu: VM started successfully, SSH server is up
2025-06-24 08:59:19.988518798 +0000 UTC qemu: Connection closed by 10.0.2.2 port 35300
2025-06-24 08:59:20.026426278 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 35316 ssh2: ECDSA SHA256:r3htoN+1DkBAkQPhqclYzvc8Rg4sGK4Qld4h7U9mv70
2025-06-24 08:59:20.035385757 +0000 UTC qemu: Connection closed by 10.0.2.2 port 35316
2025-06-24 08:59:20.062344075 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 35332 ssh2: ECDSA SHA256:r3htoN+1DkBAkQPhqclYzvc8Rg4sGK4Qld4h7U9mv70
2025-06-24 08:59:20.094383576 +0000 UTC qemu: Accepted publickey for root from 10.0.2.2 port 51910 ssh2: ECDSA SHA256:r3htoN+1DkBAkQPhqclYzvc8Rg4sGK4Qld4h7U9mv70
2025-06-24 08:59:20.100992156 +0000 UTC qemu: setting up melange cachedir: ./melange-cache/
2025-06-24 08:59:20.114164938 +0000 UTC qemu: setting up local workspace
2025-06-24 08:59:20.129744579 +0000 UTC running step "fetch"
2025-06-24 08:59:20.14240167 +0000 UTC --2025-06-24 08:59:20--  https://github.com/htop-dev/htop/releases/download/3.4.1/htop-3.4.1.tar.xz
2025-06-24 08:59:20.160729568 +0000 UTC Resolving github.com... 140.82.113.3
2025-06-24 08:59:20.187360498 +0000 UTC Connecting to github.com|140.82.113.3|:443... connected.
2025-06-24 08:59:20.3181102 +0000 UTC HTTP request sent, awaiting response... 302 Found
2025-06-24 08:59:20.3181538 +0000 UTC Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/288082909/ea3ebed0-ee53-4f19-a4c9-5f50504d0087?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20250624%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250624T085920Z&X-Amz-Expires=1800&X-Amz-Signature=6e394d25a3ea8f05af899b23b9544bf94d9f9ffe23857d66b515104ae5488ca3&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dhtop-3.4.1.tar.xz&response-content-type=application%2Foctet-stream [following]
2025-06-24 08:59:20.318159937 +0000 UTC --2025-06-24 08:59:20--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/288082909/ea3ebed0-ee53-4f19-a4c9-5f50504d0087?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=releaseassetproduction%2F20250624%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20250624T085920Z&X-Amz-Expires=1800&X-Amz-Signature=6e394d25a3ea8f05af899b23b9544bf94d9f9ffe23857d66b515104ae5488ca3&X-Amz-SignedHeaders=host&response-content-disposition=attachment%3B%20filename%3Dhtop-3.4.1.tar.xz&response-content-type=application%2Foctet-stream
2025-06-24 08:59:20.324151475 +0000 UTC Resolving objects.githubusercontent.com... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
2025-06-24 08:59:20.335781752 +0000 UTC Connecting to objects.githubusercontent.com|185.199.108.133|:443... connected.
2025-06-24 08:59:20.463200498 +0000 UTC HTTP request sent, awaiting response... 200 OK
2025-06-24 08:59:20.463276118 +0000 UTC Length: 427444 (417K) [application/octet-stream]
2025-06-24 08:59:20.463322096 +0000 UTC Saving to: 'htop-3.4.1.tar.xz'
2025-06-24 08:59:20.464796515 +0000 UTC 
2025-06-24 08:59:20.477650602 +0000 UTC      0K .......... .......... .......... .......... .......... 11% 3.42M 0s
2025-06-24 08:59:20.487635106 +0000 UTC     50K .......... .......... .......... .......... .......... 23% 4.88M 0s
2025-06-24 08:59:20.490110695 +0000 UTC    100K .......... .......... .......... .......... .......... 35% 19.9M 0s
2025-06-24 08:59:20.498731909 +0000 UTC    150K .......... .......... .......... .......... .......... 47% 5.71M 0s
2025-06-24 08:59:20.500069702 +0000 UTC    200K .......... .......... .......... .......... .......... 59% 34.2M 0s
2025-06-24 08:59:20.502289812 +0000 UTC    250K .......... .......... .......... .......... .......... 71% 22.9M 0s
2025-06-24 08:59:20.503348622 +0000 UTC    300K .......... .......... .......... .......... .......... 83% 43.2M 0s
2025-06-24 08:59:20.509097709 +0000 UTC    350K .......... .......... .......... .......... .......... 95% 8.87M 0s
2025-06-24 08:59:20.509139811 +0000 UTC    400K .......... .......                                    100% 65.7M=0.05s
2025-06-24 08:59:20.509148165 +0000 UTC 
2025-06-24 08:59:20.509237955 +0000 UTC 2025-06-24 08:59:20 (8.91 MB/s) - 'htop-3.4.1.tar.xz' saved [427444/427444]
2025-06-24 08:59:20.509252691 +0000 UTC 
2025-06-24 08:59:20.510453793 +0000 UTC fetch: Expected sha256: 904f7d4580fc11cffc7e0f06895a4789e0c1c054435752c151e812fead9f6220
2025-06-24 08:59:20.619746197 +0000 UTC autoreconf: export WARNINGS=all
2025-06-24 08:59:20.621361502 +0000 UTC autoreconf: Entering directory '.'
2025-06-24 08:59:20.622581877 +0000 UTC autoreconf: configure.ac: not using Gettext
2025-06-24 08:59:21.623379823 +0000 UTC autoreconf: running: aclocal --force 
2025-06-24 08:59:22.005600462 +0000 UTC autoreconf: configure.ac: tracing
2025-06-24 08:59:22.366415258 +0000 UTC autoreconf: configure.ac: not using Libtool
2025-06-24 08:59:22.366492927 +0000 UTC autoreconf: configure.ac: not using Intltool
2025-06-24 08:59:22.366498585 +0000 UTC autoreconf: configure.ac: not using Gtkdoc
2025-06-24 08:59:22.366503119 +0000 UTC autoreconf: running: /usr/bin/autoconf --force
2025-06-24 08:59:22.781377485 +0000 UTC autoreconf: running: /usr/bin/autoheader --force
2025-06-24 08:59:23.132157552 +0000 UTC autoreconf: running: automake --add-missing --copy --force-missing
2025-06-24 08:59:23.483757439 +0000 UTC autoreconf: Leaving directory '.'
2025-06-24 08:59:23.485404798 +0000 UTC running step "autoconf/configure"
2025-06-24 08:59:23.556947936 +0000 UTC configure: WARNING: unrecognized options: --enable-cgroup, --enable-taskstats
2025-06-24 08:59:23.59680901 +0000 UTC checking build system type... x86_64-pc-linux-gnu
2025-06-24 08:59:23.600956333 +0000 UTC checking host system type... x86_64-pc-linux-gnu
2025-06-24 08:59:23.644594368 +0000 UTC checking for a BSD-compatible install... /usr/bin/install -c
2025-06-24 08:59:23.646800177 +0000 UTC checking whether sleep supports fractional seconds... yes
2025-06-24 08:59:23.973460752 +0000 UTC checking filesystem timestamp resolution... 2
2025-06-24 08:59:23.976927695 +0000 UTC checking whether build environment is sane... yes
2025-06-24 08:59:23.986166415 +0000 UTC checking for a race-free mkdir -p... /usr/sbin/mkdir -p
2025-06-24 08:59:23.986246987 +0000 UTC checking for gawk... no
2025-06-24 08:59:23.986269209 +0000 UTC checking for mawk... no
2025-06-24 08:59:23.986327146 +0000 UTC checking for nawk... no
2025-06-24 08:59:23.986402173 +0000 UTC checking for awk... awk
2025-06-24 08:59:23.993164365 +0000 UTC checking whether make sets $(MAKE)... yes
2025-06-24 08:59:23.999348926 +0000 UTC checking whether make supports nested variables... yes
2025-06-24 08:59:24.006917663 +0000 UTC checking xargs -n works... yes
2025-06-24 08:59:24.011661717 +0000 UTC checking whether UID '0' is supported by ustar format... yes
2025-06-24 08:59:24.011686487 +0000 UTC checking whether GID '0' is supported by ustar format... yes
2025-06-24 08:59:24.03972583 +0000 UTC checking how to create a ustar tar archive... none
2025-06-24 08:59:24.048065948 +0000 UTC checking whether make supports the include directive... yes (GNU style)
2025-06-24 08:59:24.048109783 +0000 UTC checking for x86_64-pc-linux-gnu-gcc... x86_64-pc-linux-gnu-gcc
2025-06-24 08:59:24.126609424 +0000 UTC checking whether the C compiler works... yes
2025-06-24 08:59:24.126675574 +0000 UTC checking for C compiler default output file name... a.out
2025-06-24 08:59:24.178618547 +0000 UTC checking for suffix of executables... 
2025-06-24 08:59:24.231953014 +0000 UTC checking whether we are cross compiling... no
2025-06-24 08:59:24.261852466 +0000 UTC checking for suffix of object files... o
2025-06-24 08:59:24.289205723 +0000 UTC checking whether the compiler supports GNU C... yes
2025-06-24 08:59:24.317529928 +0000 UTC checking whether x86_64-pc-linux-gnu-gcc accepts -g... yes
2025-06-24 08:59:24.392755626 +0000 UTC checking for x86_64-pc-linux-gnu-gcc option to enable C11 features... none needed
2025-06-24 08:59:24.44326833 +0000 UTC checking whether x86_64-pc-linux-gnu-gcc understands -c and -o together... yes
2025-06-24 08:59:24.47740029 +0000 UTC checking dependency style of x86_64-pc-linux-gnu-gcc... gcc3
2025-06-24 08:59:24.5079511 +0000 UTC checking for stdio.h... yes
2025-06-24 08:59:24.547151509 +0000 UTC checking for stdlib.h... yes
2025-06-24 08:59:24.585859694 +0000 UTC checking for string.h... yes
2025-06-24 08:59:24.624053449 +0000 UTC checking for inttypes.h... yes
2025-06-24 08:59:24.661532689 +0000 UTC checking for stdint.h... yes
2025-06-24 08:59:24.696811304 +0000 UTC checking for strings.h... yes
2025-06-24 08:59:24.733673382 +0000 UTC checking for sys/stat.h... yes
2025-06-24 08:59:24.770961212 +0000 UTC checking for sys/types.h... yes
2025-06-24 08:59:24.816899515 +0000 UTC checking for unistd.h... yes
2025-06-24 08:59:24.865412836 +0000 UTC checking for wchar.h... yes
2025-06-24 08:59:24.910234122 +0000 UTC checking for minix/config.h... no
2025-06-24 08:59:24.979126253 +0000 UTC checking whether it is safe to define __EXTENSIONS__... yes
2025-06-24 08:59:25.013643026 +0000 UTC checking whether _XOPEN_SOURCE should be defined... no
2025-06-24 08:59:25.07331912 +0000 UTC checking for x86_64-pc-linux-gnu-gcc option to enable large file support... none needed
2025-06-24 08:59:25.073406601 +0000 UTC checking for x86_64-pc-linux-gnu-gcc... (cached) x86_64-pc-linux-gnu-gcc
2025-06-24 08:59:25.107521011 +0000 UTC checking whether the compiler supports GNU C... (cached) yes
2025-06-24 08:59:25.107565499 +0000 UTC checking whether x86_64-pc-linux-gnu-gcc accepts -g... (cached) yes
2025-06-24 08:59:25.107627575 +0000 UTC checking for x86_64-pc-linux-gnu-gcc option to enable C11 features... (cached) none needed
2025-06-24 08:59:25.107663932 +0000 UTC checking whether x86_64-pc-linux-gnu-gcc understands -c and -o together... (cached) yes
2025-06-24 08:59:25.1078009 +0000 UTC checking dependency style of x86_64-pc-linux-gnu-gcc... (cached) gcc3
2025-06-24 08:59:25.142248092 +0000 UTC checking for dirent.h that defines DIR... yes
2025-06-24 08:59:25.189943459 +0000 UTC checking for library containing opendir... none required
2025-06-24 08:59:25.191458887 +0000 UTC checking for stdlib.h... (cached) yes
2025-06-24 08:59:25.196081648 +0000 UTC checking for string.h... (cached) yes
2025-06-24 08:59:25.200719607 +0000 UTC checking for strings.h... (cached) yes
2025-06-24 08:59:25.256982856 +0000 UTC checking for sys/param.h... yes
2025-06-24 08:59:25.31014726 +0000 UTC checking for sys/time.h... yes
2025-06-24 08:59:25.360062081 +0000 UTC checking for sys/utsname.h... yes
2025-06-24 08:59:25.364204222 +0000 UTC checking for unistd.h... (cached) yes
2025-06-24 08:59:25.415962984 +0000 UTC checking for sys/mkdev.h... no
2025-06-24 08:59:25.461428451 +0000 UTC checking for sys/sysmacros.h... yes
2025-06-24 08:59:25.50987112 +0000 UTC checking for execinfo.h... yes
2025-06-24 08:59:25.570386071 +0000 UTC checking for mbstate_t... yes
2025-06-24 08:59:25.677679445 +0000 UTC checking for mode_t... yes
2025-06-24 08:59:25.822104794 +0000 UTC checking for off_t... yes
2025-06-24 08:59:25.924079769 +0000 UTC checking for pid_t... yes
2025-06-24 08:59:26.026083697 +0000 UTC checking for size_t... yes
2025-06-24 08:59:26.129788364 +0000 UTC checking for ssize_t... yes
2025-06-24 08:59:26.231139788 +0000 UTC checking for uid_t... yes
2025-06-24 08:59:26.332153673 +0000 UTC checking for gid_t... yes
2025-06-24 08:59:26.387154784 +0000 UTC checking for uint8_t... yes
2025-06-24 08:59:26.445207868 +0000 UTC checking for uint16_t... yes
2025-06-24 08:59:26.49937159 +0000 UTC checking for uint32_t... yes
2025-06-24 08:59:26.552789298 +0000 UTC checking for uint64_t... yes
2025-06-24 08:59:26.578646151 +0000 UTC checking for alloc_size... yes
2025-06-24 08:59:26.602000719 +0000 UTC checking for access... yes
2025-06-24 08:59:26.623397239 +0000 UTC checking for nonnull... yes
2025-06-24 08:59:26.644403217 +0000 UTC checking for returns_nonnull... yes
2025-06-24 08:59:26.721945153 +0000 UTC checking for NaN support... yes
2025-06-24 08:59:26.749333998 +0000 UTC checking for __builtin_ctz... yes
2025-06-24 08:59:26.856166608 +0000 UTC checking for library containing ceil... -lm
2025-06-24 08:59:26.908712209 +0000 UTC checking for library containing dlopen... none required
2025-06-24 08:59:26.958703904 +0000 UTC checking for library containing clock_gettime... none required
2025-06-24 08:59:27.013365481 +0000 UTC checking for clock_gettime... yes
2025-06-24 08:59:27.067058032 +0000 UTC checking for dladdr... yes
2025-06-24 08:59:27.121004469 +0000 UTC checking for faccessat... yes
2025-06-24 08:59:27.171191207 +0000 UTC checking for fstatat... yes
2025-06-24 08:59:27.231548068 +0000 UTC checking for host_get_clock_service... no
2025-06-24 08:59:27.285963591 +0000 UTC checking for memfd_create... yes
2025-06-24 08:59:27.335528921 +0000 UTC checking for openat... yes
2025-06-24 08:59:27.394908006 +0000 UTC checking for readlinkat... yes
2025-06-24 08:59:27.449190981 +0000 UTC checking for sched_getscheduler... yes
2025-06-24 08:59:27.497456519 +0000 UTC checking for sched_setscheduler... yes
2025-06-24 08:59:27.548017583 +0000 UTC checking for strchrnul... yes
2025-06-24 08:59:27.549323722 +0000 UTC checking for x86_64-pc-linux-gnu-pkg-config... no
2025-06-24 08:59:27.5494167 +0000 UTC checking for pkg-config... /usr/sbin/pkg-config
2025-06-24 08:59:27.550505753 +0000 UTC checking pkg-config is at least version 0.9.0... yes
2025-06-24 08:59:27.560815734 +0000 UTC found ncursesw information through /usr/sbin/pkg-config
2025-06-24 08:59:27.611495546 +0000 UTC checking for keypad in -lncursesw -ltinfo... yes
2025-06-24 08:59:27.660138361 +0000 UTC checking for doupdate in -lncursesw -ltinfo... yes
2025-06-24 08:59:27.710947445 +0000 UTC checking for wadd_wchnstr in -lncursesw -ltinfo... yes
2025-06-24 08:59:27.762045645 +0000 UTC checking for ncursesw/curses.h... no
2025-06-24 08:59:27.808869312 +0000 UTC checking for ncurses/ncurses.h... no
2025-06-24 08:59:27.85521396 +0000 UTC checking for ncurses/curses.h... no
2025-06-24 08:59:27.916497492 +0000 UTC configure: error: can not find required ncurses header file
2025-06-24 08:59:27.916539718 +0000 UTC checking for ncurses.h... no
2025-06-24 08:59:27.939269871 +0000 UTC Failed wait for session running: "/bin/sh -c 'set -e 
[ -d '\''/home/build'\'' ] || mkdir -p '\''/home/build'\''
cd '\''/home/build'\''
cd .
if [[ ! -f ./configure && -f ./configure.ac ]]; then
	autoreconf -vfi
fi
./configure \
	--host=x86_64-pc-linux-gnu \
	--build=x86_64-pc-linux-gnu \
	--prefix=/usr \
	--sysconfdir=/etc \
	--libdir=/usr/lib \
	--mandir=/usr/share/man \
	--infodir=/usr/share/info \
	--localstatedir=/var \
	--prefix="/usr" \
	--sysconfdir="/etc" \
	--mandir="/usr/share/man" \
	--localstatedir="/var" \
	--enable-cgroup \
	--enable-capabilities \
	--enable-delayacct \
	--enable-taskstats

exit 0'": Process exited with status 1
2025-06-24 08:59:27.939354211 +0000 UTC qemu: sending shutdown signal
2025-06-24 08:59:27.945386087 +0000 UTC failed to build package: unable to run package htop pipeline: unable to run pipeline: unable to run pipeline: Process exited with status 1
2025-06-24 08:59:27.997285436 +0000 UTC failed to build: command melange failed: exit status 1
2025-06-24 08:59:27.997469696 +0000 UTC build failed: command melange failed: exit status 1
```

--- a/htop.yaml
+++ b/htop.yaml
@@ -1,7 +1,7 @@
 package:
   name: htop
   version: "3.4.1"
-  epoch: 1
+  epoch: 2
   description: "A cross-platform interactive process viewer."
   copyright:
     - license: GPL-2.0-or-later
@@ -17,6 +17,7 @@ environment:
       - libnl3-dev
       - linux-headers
       - ncurses-dev
+      - ncurses-static
       - pkgconf-dev
 
 pipeline:

